src/beautifiers/verilog-mode/index.coffee

```diff
@@ -0,0 +1,63 @@
+###
+Requires http://golang.org/cmd/gofmt/
+###
+
+"use strict"
+Beautifier = require('../beautifier')
+path = require("path")
+
+module.exports = class EmacsVerilogMode extends Beautifier
+  name: "Emacs Verilog Mode"
+  link: "https://www.veripool.org/projects/verilog-mode/"
+  isPreInstalled: false
+  executables: [
+    {
+      name: "Emacs"
+      cmd: "emacs"
+      homepage: "https://www.gnu.org/software/emacs/"
+      installation: "https://www.gnu.org/software/emacs/"
+      version: {
+        parse: (text) -> text.match(/Emacs (\d+\.\d+\.\d+)/)[1]
+      }
+    }
+  ]
+
+  options: {
+    Verilog: true
+  }
+
+  beautify: (text, language, options) ->
+    emacs = @exe("emacs")
+
+    emacs_path = options.emacs_path
+    emacs_script_path = options.verilog_emacs_script_path
+
+    if not emacs_script_path
+      emacs_script_path = path.resolve(__dirname, "verilog-mode.el")
+
+    @debug('verilog-beautifier', 'emacs script path: ' + emacs_script_path)
+
+    tempFile = @tempFile("input", text)
+
+    args = [
+      "--batch"
+      tempFile
+      "-l"
+      emacs_script_path
+      "-f"
+      "verilog-mode"
+      "-f"
+      "verilog-batch-indent"
+      ]
+
+    if emacs_path
+      @deprecateOptionForExecutable("Emacs", "emacs_path", "Path")
```

Comment:
No need for deprecating an option on a *new* beautifier.
Looks like this was copied from https://github.com/Glavin001/atom-beautify/blob/49982c6ce7b0c8f66042c2eddbc49d1441944023/src/beautifiers/fortran-beautifier/index.coffee#L50

Lines 53 through 59 can be deleted. The deprecation was because the Executables feature was new to to "Fortran Beautifier": https://github.com/Glavin001/atom-beautify/blob/49982c6ce7b0c8f66042c2eddbc49d1441944023/src/beautifiers/fortran-beautifier/index.coffee#L12-L22